Extensions/bezier/bezier.lisp

```diff
@@ -79,62 +79,47 @@
   (- (* (realpart z) (point-y v))
      (* (imagpart z) (point-x v))))
 
-(defclass bezier-design (design) 
-  ((%or :accessor original-region :initform nil)))
+(defclass bezier-design (climi::design)
+  ((%or :accessor original-region :initform nil)
```

Comment:
design is exported from `CLIM` package (protocol class), we shouldn't access symbols in `CLIM-INTERNALS` unless ineviatable / needed.